CompilerSource/backend/GameData.cpp

```diff
@@ -71,6 +256,24 @@ SpriteData::SpriteData(const ::Sprite &sprite):
     image_data.emplace_back(sprite.subImages[i].image);
 }
 
+SoundData::SoundData(const buffers::resources::Sound &q, const std::string& name):
+  buffers::resources::Sound(q), name(name) {
+  // Open sound
+  FILE *afile = fopen(q.data().c_str(),"rb");
+  if (!afile)
```

Comment:
Same here.

Reply:
Ok: 8f1b70790e33e057859be38918b67c9d6a731d2c

You'll need to look that over, see reply above.